app/views/surveys/new.html.haml

```diff
@@ -0,0 +1,3 @@
+%h1 Add a New Survey
+%div.review-container= render 'form'
```

Comment:
`%div.review-container` can be written as `.review-container` since `%div` is implicit